src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3567,22 +3615,18 @@ void InitIntSolarDistribution(EnergyPlusData &state)
             int const ConstrNum = Surface(SurfNum).Construction;
 
             Real64 AbsIntSurf = state.dataConstruction->Construct(ConstrNum).InsideAbsorpSolar; // Inside opaque surface solar absorptance
-            Real64 AbsIntSurfVis =
-                state.dataConstruction->Construct(ConstrNum).InsideAbsorpSolar; // Inside opaque surface visible absorptance to fix CR 8695 change to
-                                                                                // this = Construct(ConstrNum)%InsideAbsorpVis
-            Real64 HMovInsul = 0.0; // Solar absorptance of outermost layer (or movable insulation if present)
-            Real64 AbsInt;          // Inside opaque surface solar absorptance
-            Real64 AbsExt;          // Solar absorptance of outermost layer (or movable insulation if present)
-            int RoughIndexMovInsul; // Roughness index of movable insulation
-            if (Surface(SurfNum).MaterialMovInsulInt > 0) EvalInsideMovableInsulation(state, SurfNum, HMovInsul, AbsInt);
-            if (HMovInsul > 0.0) AbsIntSurf = AbsInt;
+            Real64 AbsIntSurfVis = state.dataConstruction->Construct(ConstrNum).InsideAbsorpSolar; // Inside opaque surface visible absorptance to fix CR 8695 change to
+
+            if (state.dataSurface->SurfMovInsulHInt(SurfNum) > 0) {
+                AbsIntSurf = state.dataSurface->SurfMovInsulAbsInt(SurfNum);
+            }
+
             state.dataHeatBalSurf->SurfOpaqQRadSWInAbs(SurfNum) += state.dataHeatBal->QS(solEnclosureNum) * AbsIntSurf;
             state.dataHeatBalSurf->SurfOpaqQRadSWLightsInAbs(SurfNum) += state.dataHeatBal->QSLights(solEnclosureNum) * AbsIntSurfVis;
 
             // Calculate absorbed solar on outside if movable exterior insulation in place
-            HMovInsul = 0.0;
-            if (Surface(SurfNum).MaterialMovInsulExt > 0) EvalOutsideMovableInsulation(state, SurfNum, HMovInsul, RoughIndexMovInsul, AbsExt);
-            if (HMovInsul > 0) { // Movable outside insulation in place
+            if (state.dataSurface->SurfMovInsulHExt(SurfNum) > 0) { // Movable outside insulation in place
```

Comment:
Because exterior movable insulation doesn't have a `Present` variable like inside movable insulation, `SurfMovInsulHExt >0` is being used to test for presence. In the old `EvalOutsideMovableInsulation` function, this was reset to zero when `MovInsulSchedVal <= 0.0` but I don't see that in the new version of this function.

Reply:
We should probably implement inside and outside `MovableInsulation` in similar ways, no? Consistency is the hobgoblin of little minds. And of Amir.

Reply:
I added `SurfMovInsulExtPresent` and used the boolean arrays in checking presence for both inside and outside surface.